packages/core/components/DraggableComponent/index.tsx

```diff
@@ -163,12 +180,17 @@ export const DraggableComponent = ({
                 }}
               >
                 <CustomActionBar label={label}>
```

Comment:
I think we need to adjust the visuals of the ActionBar when there are no actions inside, as the current UI collapses down with poor spacing and an unnecessary separator.

<img width="229" alt="image" src="https://github.com/user-attachments/assets/1854a75e-2e4c-403d-a02c-d8db9cc5d403">

Additionally, we should provide some kind of indication that the component cannot be dragged.

I'm happy to do these as a follow up.

Reply:
Ahh yes, I did the Action Bar already but I think it was in the original PR we didn't merge..

What are you thinking as an indication that the component cannot be dragged

Reply:
I think for now perhaps just changing the pointer to `pointer` instead of `grab` is enough.

I plan on adding grab handles to the action bar in #556, which we could then add a little 🔒 illustration to.